modules/http/src/main/java/org/glassfish/grizzly/http/HttpServerFilter.java

```diff
@@ -1195,6 +1204,24 @@ private void sendBadRequestResponse(final FilterChainContext ctx,
         commitAndCloseAsError(ctx, response);
     }
 
+    private void sendEntityTooLargeResponse(final FilterChainContext ctx,
+                                        final HttpResponsePacket response) {
+        if (response.getHttpStatus().getStatusCode() < 400) {
+            // 413 - Request Entity Too Large
+            HttpStatus.REQUEST_ENTITY_TOO_LARGE_413.setValues(response);
+        }
+        commitAndCloseAsError(ctx, response);
```

Comment:
I think this could result in double closing and committing.

Reply:
I copy the code from the function sendBadRequestResponse.

Reply:
Ok